Feat/about

naomi-eliasar opened the pull request:
- Created person cards for about page
- Made article post on blog roll clickable to see entire article
- Gave articles on blog roll a little different styling than the entire article

drikusroor commented on web/src/index.css:
```diff
@@ -20,4 +20,12 @@
+
+  .h2 {
+      @apply text-lg font-semibold;
+    }
```
I see this is indented. It's not a big issue but an autoformatter plugin such as `prettier` in your vs code might help here.

drikusroor commented on web/src/pages/AboutPage/AboutPage.tsx:
```diff
@@ -1,19 +1,44 @@
+      <div className="grid grid-cols-2 grid-rows-3 gap-4 pb-4">
+        <Person
+          name="Adriana"
+          quote="えーどりあな"
```
_Technically_, I think we should write (especially foreign) names using katakana and not hiragana 😅

drikusroor commented on web/src/pages/AboutPage/AboutPage.tsx:
```diff
@@ -1,19 +1,44 @@
+        <Person
+          name="Adriana"
+          quote="えーどりあな"
+          story="Insert story about yourself here"
```
Perhaps for now we can use empty strings here because no story might look better than "Insert story about yourself here". On the other hand, we aren't really "live" so it also doesn't matter that much.

drikusroor commented on web/src/pages/AboutPage/AboutPage.tsx:
```diff
@@ -1,19 +1,44 @@
+          name="Adriana"
+          quote="えーどりあな"
+          story="Insert story about yourself here"
+          imgSrc="https://media.licdn.com/dms/image/D4E03AQErJ1uyanV3Hg/profile-displayphoto-shrink_200_200/0/1685713399087?e=1692230400&v=beta&t=SEH4cQD6g_GpZU3_RdsGtfFJDSCo8-CH2diebHYmhmQ"
```
I see you have used some place to upload the images, am I right? Will these images be up there for the coming half year and can we trust its reliability? It might be better to just put the images in the `web/public/images` (you might to create the images folder manually) as we have direct control over it.

naomi-eliasar replied:
No, I just used the pictures from linkedin for now. But I will create the images folder!

drikusroor commented on web/src/layouts/BlogLayout/BlogLayout.tsx:
```diff
@@ -43,13 +41,13 @@ const BlogLayout = ({ children }: BlogLayoutProps) => {
             <li>
               <Link className="rw-button" to={routes.about()}>
-                About
+                Reisgezelschap
```
Nice, or how about Reisgenootschap, to make it more LotR-ish? 🙀

drikusroor commented on web/src/layouts/BlogLayout/BlogLayout.tsx:
```diff
@@ -43,13 +41,13 @@ const BlogLayout = ({ children }: BlogLayoutProps) => {
         </nav>
       </header>
-      <main className="mx-auto max-w-xl">{children}</main>
+      <main className="max-w-screen mx-auto p-4">{children}</main>
```
Was it a conscious decision to use the whole width of the screen for listing the posts? It will look like this on my external monitor: 😅 
![afbeelding](https://github.com/drikusroor/ohayo-goededagu/assets/8208970/cf65b760-55b9-401d-bcb6-6274a9e3b33f)

drikusroor reviewed (changes requested):
Using katakana instead of hiragana is fairly important I'd say. And I also think it'd be better to keep the images in the repository instead of on media.licdn.com. But for the rest I give you full freedom to decide what you want to do about with my feedback.

naomi-eliasar commented:
> Using katakana instead of hiragana is fairly important I'd say. And I also think it'd be better to keep the images in the repository instead of on media.licdn.com. But for the rest I give you full freedom to decide what you want to do about with my feedback.

I've changed the names to katakana, except for mine 😄 
Also implemented pretty much all your other comments, as I agreed

drikusroor reviewed (approved):
Yesss, looks fine now
